pkg/ebpf/c/offset-guess.c

```diff
@@ -146,15 +147,21 @@ int kprobe__ip_make_skb(struct pt_regs* ctx) {
 }
 
 /* Used exclusively for offset guessing */
-SEC("kprobe/tcp_get_info")
-int kprobe__tcp_get_info(struct pt_regs* ctx) {
+SEC("kprobe/tcp_getsockopt")
+int kprobe__tcp_getsockopt(struct pt_regs* ctx) {
+    int level = (int)PT_REGS_PARM2(ctx);
```

Comment:
what does the `level` option control? I couldn't easily tell from looking at the source

Reply:
It is the `level` argument from the `getsockopt` system call. https://linux.die.net/man/2/getsockopt

The filters I'm using match what we call `getsockopt` with:
https://github.com/DataDog/datadog-agent/blob/fa0db67f8ed9b4328b453c7157020be4223a9bcf/pkg/ebpf/offsetguess.go#L672